internal/cmd/gocdk/biomeutil.go

```diff
@@ -41,27 +41,32 @@ func biomesRootDir(moduleRoot string) string {
 	return filepath.Join(moduleRoot, "biomes")
 }
 
-// biomeDir returns the path to the named biome.
-func biomeDir(moduleRoot, name string) string {
+// biomeDir returns the path to the named biome. It returns an error if the
+// named biome does not exist.
+func biomeDir(moduleRoot, name string) (string, error) {
+	dir := filepath.Join(biomesRootDir(moduleRoot), name)
+	if _, err := os.Stat(filepath.Join(dir, biomeConfigFileName)); err != nil {
+		// TODO(light): Wrap error for formatting chain but not unwrap chain.
+		return "", &biomeNotFoundError{moduleRoot: moduleRoot, biome: name, frame: xerrors.Caller(0), detail: err}
+	}
+	return dir, nil
+}
+
+// newBiomeDir returns the path to the named biome, without checking to see
+// if the biome exists.
+func newBiomeDir(moduleRoot, name string) string {
```

Comment:
The name `newBiomeDir` here seems confusing, as it implies it would create the biome directory.

I see two options:
1) Call `os.MkdirAll` in this function.
2) Make `biomeDir`'s first return the correct path, and just ignore the error where appropriate.

While option 2 is slightly inefficient (1 unneeded syscall), IMO it makes the intent of the caller more clear.

Reply:
Did #2.